specification/schemas/BaseAddress.json

```diff
@@ -0,0 +1,86 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "street_1": {
+      "type": "string",
+      "example": "Baker Street"
+    },
+    "street_2": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "example": "Marylebone"
+    },
+    "street_number": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "example": 221
+    },
+    "street_number_suffix": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "example": "B"
+    },
+    "postal_code": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "example": "NW1 6XE"
+    },
+    "city": {
+      "type": "string",
+      "example": "London"
+    },
+    "region_code": {
+      "$ref": "#/components/schemas/RegionCode"
+    },
+    "country_code": {
+      "$ref": "#/components/schemas/CountryCode"
+    },
+    "first_name": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "example": "Sherlock"
+    },
+    "last_name": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "example": "Holmes"
+    },
+    "company": {
+      "type": [
+        "string",
+        "null"
```

Comment:
In other parts you're using `"oneOf"` instead of a type array, is there a reason that it's different here?

Reply:
In other parts some cases where I use a `"oneOf"`, it's because that attribute has enum values. In order for null to be accepted as a value in an enum, you need to add it to the enum. In the UI however, that would show up as: `"enum": [ first, second, ].` Notice the comma with empty space behind it in the array. That's why I chose for oneOf there. In general, I prefer to use the type array for nullable values, so that's why I did it differently here, because it's not an issue here. It is a bit inconsistent. I agree.